### ISSUE TITLE
Slack integration does not work for orgs with SAML and IP restrictions

### ISSUE THREAD

**Issue by schosterbarak** [integrations/slack#1271]
**Describe the bug**
Slack integration does not work for orgs with SAML and IP restrictions
**To Reproduce**
1. Go to slack and try to subscribe public github repo that you own in org with SAML and IP restrictions
**Expected behavior**
a working notification system in slack for public repos

**Comment by quaelin** [comment]
Oh no!  We just enabled SAML auth in Github, and this integration was the first casualty.  :(

Please fix, we miss our GitHub notifs!

**Comment by yermulnik** [comment]
+1, please take a look into how this can be done

**Comment by smridge** [comment]
+1 I'm experiencing this issue as well.

**Comment by quaelin** [comment]
n/m, got it working by hitting `Configure SSO` on my ssh key [here](https://github.com/settings/keys) and then re-authing the plugin in Slack.

**Comment by smridge** [comment]
> n/m, got it working by hitting `Configure SSO` on my ssh key [here](https://github.com/settings/keys) and then re-authing the plugin in Slack.

Ah the re-authing `/github signin` was key to this. Thank you @quaelin !

**Comment by yermulnik** [comment]
Yep, this option is missing from `/github help` output which makes a lot of confusion :shrug: 
Btw keep in mind this integration can handle only one single GH account association: https://github.com/integrations/slack/issues/1324
